closure/goog/i18n/numberformat.js

```diff
@@ -580,6 +588,140 @@ goog.i18n.NumberFormat.prototype.roundNumber_ = function(number) {
 
 
 /**
+ * This function is called when the length of the repeating digits is at least
```

Comment:
I'm noticing this method description doesn't really describe *what* this method does. It says it's called, but doesn't say what its operations are and what it returns.